### ISSUE TITLE
[build] Fix handlebars warnings on yarn build

### ISSUE THREAD

**Issue by mrseanryan** [mrseanryan/tslint-folders#34]
```
yarn build
```

```
Handlebars: Access has been denied to resolve the property "hasComment" because it is not an "own property" of its parent.
You can add a runtime option to disable the check or this warning:
See https://handlebarsjs.com/api-reference/runtime-options.html#options-to-control-prototype-access for details
Handlebars: Access has been denied to resolve the property "hasGetterOrSetter" because it is not an "own property" of its parent.
You can add a runtime option to disable the check or this warning:
See https://handlebarsjs.com/api-reference/runtime-options.html#options-to-control-prototype-access for detail

...
...
```

**Comment by PFight** [comment]
> From version 4.6.0 on, Handlebars forbids accessing prototype properties and methods of the context object by default. 

You should pass[ runtime property](https://handlebarsjs.com/api-reference/runtime-options.html#options-to-control-prototype-access) when run handlebars, or do something like:

npm i -D handlebars@4.5.0

Note, that this warning can cause great problems in result.
